observation_portal/accounts/forms.py

```diff
@@ -12,18 +12,19 @@ class CustomRegistrationForm(RegistrationFormTermsOfService, RegistrationFormUni
     last_name = forms.CharField(max_length=200)
     institution = forms.CharField(max_length=200)
     title = forms.CharField(max_length=200)
-    education_user = forms.BooleanField(required=False)
+    simple_interface = forms.BooleanField(label="Use Basic Mode", help_text="Hide advanced fields when making an observation request.", required=False)
```

Comment:
The same help text is also written on line 27. Does it need to be set in these two different places?

Reply:
I've removed the help_text in Meta as that version is not being used (there is help_text for `username` which doesn't get displayed too).